data/json/items/comestibles/dairy.json

```
@@ -472,5 +472,72 @@
     "vitamins": [ [ "calcium", 7 ] ],
     "flags": [ "NUTRIENT_OVERRIDE" ],
     "fun": 1
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "tiramisu",
+    "name": { "str_sp": "tiramisù" },
+    "weight": "150 g",
+    "color": "brown",
+    "spoils_in": "14 days",
+    "container": "casserole",
+    "comestible_type": "FOOD",
+    "symbol": "t",
+    "quench": -2,
+    "healthy": -1,
+    "calories": 500,
+    "description": "It is an Italian spoon dessert based on savoiardi soaked in coffee and covered with a cream composed of mascarpone and eggs.",
+    "price": 100,
+    "price_postapoc": 300,
+    "material": [ "wheat", "milk" ],
+    "primary_material": "milk",
+    "volume": "150 ml",
+    "flags": [ "EDIBLE_FROZEN" ],
+    "charges": 4,
+    "vitamins": [ [ "calcium", 15 ], [ "iron", 2 ] ],
+    "fun": 8
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "mascarpone",
+    "name": { "str_sp": "mascarpone" },
+    "weight": "250 g",
+    "color": "white",
+    "spoils_in": "12 days",
+    "container": "cup_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 450,
+    "description": "a portion of mascarpone, a dairy product obtained from the processing of cream and citric acid, a typical Italian spreadable cheese.",
+    "price": 200,
+    "price_postapoc": 250,
+    "material": [ "milk" ],
+    "volume": "180 ml",
+    "phase": "solid",
+    "flags": [ "EDIBLE_FROZEN" ],
+    "fun": -1,
+    "vitamins": [ [ "calcium", 9 ], [ "iron", 2 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pecorino_cheese",
+    "name": { "str_sp": "pecorino" },
+    "weight": "100 g",
+    "color": "white",
+    "spoils_in": "180 days",
+    "container": "wrapper",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 350,
+    "description": "a slice of pecorino, famous Italian hard cheese used in pasta carbonara.",
```

```suggestion
    "description": "A slice of pecorino, famous Italian hard cheese used in pasta carbonara.",
```
~~I use parmesan cuz I cheap~~